File: API/DTOs/DoctorDto.cs
using System;
using System.Collections.Generic;

namespace API.DTOs
{
    public class DoctorDto
    {
        public int Id {get; set;}
        public string FirstName { get; set; }
        public string SecondName { get; set; }
        public string Email { get; set; }
        public string DateOfBirth { get; set; }
        public int Age {get;set;}
        public ICollection<StudiesAndExperienceDto> StudiesAndExperience { get; set; }
        public ICollection<WorkDayDto> WorkDays { get; set; }
    }
}

File: client/src/app/_models/doctorDto.ts
import { PhotoDto } from "./photoDto";
import { StudiesAndExperience } from "./studiesAndExperienceDto";
import { WorkDayDto } from "./workDayDto";

export interface DoctorDto {
    firstName: string;
    secondName: string;
    email: string;
    dateOfBirth: Date;
    age: number;
    mainPhotoUrl: string;
    studiesAndExperience: StudiesAndExperience[];
    workDays: WorkDayDto[];
    photos: PhotoDto[];
  }